go/api/glide_cluster_client.go

```diff
@@ -16,3 +20,37 @@ func NewGlideClusterClient(config *GlideClusterClientConfiguration) (*GlideClust
 
 	return &GlideClusterClient{client}, nil
 }
+
+// CustomCommand executes a single command, specified by args, without checking inputs. Every part of the command, including
+// the command name and subcommands, should be added as a separate value in args. The returning value depends on the executed
+// command.
+//
+// The command will be routed automatically based on the passed command's default request policy.
+//
+// See [Valkey GLIDE Wiki] for details on the restrictions and limitations of the custom command API.
+//
+// This function should only be used for single-response commands. Commands that don't return complete response and awaits
+// (such as SUBSCRIBE), or that return potentially more than a single response (such as XREAD), or that change the client's
+// behavior (such as entering pub/sub mode on RESP2 connections) shouldn't be called using this function.
+//
+// Parameters:
+//
+//	args - Arguments for the custom command including the command name.
+//
+// Return value:
+//
+//	The returned value for the custom command.
+//
+// For example:
+//
+//	result, err := client.CustomCommand([]string{"ping"})
+//	result.(string): "PONG"
+//
+// [Valkey GLIDE Wiki]: https://github.com/valkey-io/valkey-glide/wiki/General-Concepts#custom-command
+func (client *GlideClusterClient) CustomCommand(args []string) (interface{}, error) {
```

Comment:
Don't we want to handle the `ClusterResponse` support here? Ref: https://github.com/valkey-io/valkey-glide/blob/main/java/client/src/main/java/glide/api/commands/GenericClusterCommands.java#L37

Reply:
I plan to add it in a follow up PR.
We need to unblock #2801 with current change.

Reply:
I decided to try including this into this PR, see 8c6fac28a